trie/trieStorageManager.go

```diff
@@ -580,6 +580,8 @@ func (tsm *trieStorageManager) RemoveFromCheckpointHashesHolder(hash []byte) {
 	tsm.storageOperationMutex.Lock()
 	defer tsm.storageOperationMutex.Unlock()
 
+	log.Trace("trie storage manager: remove", "hash", hash)
```

Comment:
update log message to include RemoveFromCheckpointHashesHolder or something similar?

Reply:
Done.